stubs/python-xlib/Xlib/protocol/display.pyi

```diff
@@ -0,0 +1,120 @@
+from _typeshed import _BufferWithLen
+from socket import socket
+from typing import TypeVar, overload
+from typing_extensions import Literal
+
+from Xlib import error
+from Xlib._typing import ErrorHandler
+from Xlib.display import _resource_baseclasses, _ResourceBaseClass  # pyright: ignore[reportPrivateUsage]
```

Comment:
Do we have `reportPrivateUsage` enabled in our pyright config?

Reply:
I have a few `# pyright: ignore` leftovers that need to be removed.